core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -398,6 +399,8 @@
         "alluxio.underfs.oss.connection.timeout.ms";
     public static final String UNDERFS_OSS_CONNECT_TTL = "alluxio.underfs.oss.connection.ttl";
     public static final String UNDERFS_OSS_SOCKET_TIMEOUT = "alluxio.underfs.oss.socket.timeout.ms";
+    public static final String UNDERFS_S3A_ENABLE_ACL =
+        "alluxio.underfs.s3a.enable.acl";
```

Comment:
The convention in Alluxio is to use 100 characters per line. You can find the style guide here: http://www.alluxio.org/docs/master/en/Contributing-to-Alluxio.html#coding-style